examples/basic-otlp/src/main.rs

```diff
@@ -106,7 +102,7 @@ async fn main() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
     tokio::time::sleep(Duration::from_secs(60)).await;
 
     shutdown_tracer_provider();
-    metrics_controller.stop(&cx)?;
+    meter_provider.shutdown()?;
```

Comment:
the comment above indicate that we need to wait for 1 minutes. Is that needed? Since we shutdown provider here, and shutdown should be flushing out any pending metrics to exporter, and exporter should also be flushing out anything to OTLP....

Reply:
That was from before, but happy to remove it as it doesn't appear to be necessary.